File: fds-backend/user-management-service/src/main/java/com/yokke/usermanagement/transactions/fds_parameter/FdsParameterDto.java
package com.yokke.usermanagement.transactions.fds_parameter;

import lombok.*;

import java.io.Serializable;

/**
 * DTO for {@link FdsParameter}
 */
@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class FdsParameterDto implements Serializable {
    String fdsParameterId;
    String fdsParameterKey;
    String fdsParameterValue;
    String fdsParameterCategory;
}

File: fds-dashboard/src/packages/user-management-service/src/fds-parameter/dto/fds-parameter.dto.ts
import type { BasePageableDto } from "@/packages/base/src/dto/base-pageable.dto";

export type FdsParameterDto = {
  fdsParameterId: string;
  fdsParameterKey: string;
  fdsParameterValue: string;
  fdsParameterCategory: string;
};

export type FdsParameterPageableDto = {
  content: FdsParameterDto[];
} & BasePageableDto;
